storage/src/state/tests.rs

```diff
@@ -346,3 +347,29 @@ fn test_transaction_fees() {
     assert_eq!(new_coinbase_record.value(), expected_block_reward);
     assert_eq!(output_record.value(), amount);
 }
+
+#[test]
+fn test_get_blocks_iterator() {
+    // Initialize a new temporary directory.
+    let directory = temp_dir();
+
+    // Initialize an empty ledger.
+    let ledger_state = LedgerState::open_writer::<RocksDB, _>(directory.clone()).expect("Failed to initialize ledger");
+
+    // Read the test blocks;
+    // note: they don't include the genesis block, as it's always available when creating a ledger.
+    let test_blocks = fs::read("benches/blocks_1").unwrap_or_else(|_| panic!("Missing the test blocks file"));
+    let blocks: Vec<Block<Testnet2>> = bincode::deserialize(&test_blocks).expect("Failed to deserialize a block dump");
+
+    // Load a test block into the ledger.
+    ledger_state.add_next_block(&blocks[0]).expect("Failed to add a test block");
+    let blocks_result: Vec<_> = ledger_state
+        .get_blocks(0, ledger_state.latest_block_height() + 1)
+        .unwrap()
+        .filter_map(|block_result| block_result.ok())
+        .collect();
+
+    drop(ledger_state);
+
+    assert_eq!(blocks_result, vec![Testnet2::genesis_block().clone(), blocks[0].clone()]);
+}
```

Comment:
Shouldn't be here a last EOL?

Reply:
The EOL is there, GitHub deletes that empty line at the end in the PRs.